lib/web/ca_export.go

```diff
@@ -119,3 +126,19 @@ func (h *Handler) authExportPublicZip(
 	http.ServeContent(w, r, zipName, now, bytes.NewReader(out.Bytes()))
 	return nil
 }
+
+func (h *Handler) authExportPublicJSON(
+	w http.ResponseWriter,
+	r *http.Request,
+	authorities []*client.ExportedAuthority,
+) error {
+	marshalledAuthorities, err := json.Marshal(authorities)
+	if err != nil {
+		return trace.Wrap(err, "failed to JSON marshal authorities")
+	}
+
+	// File name is not critical here. It is only used by `ServeContent` to determine the value of the
+	// `Content-Type` header.
+	http.ServeContent(w, r, "export.json", time.Now(), bytes.NewReader(marshalledAuthorities))
```

Comment:
Should we set the `Content-Disposition` header here too?

Reply:
We can, though I don't think that it is critical, given that (unlike zip files) browsers can render this output without issue. We don't currently set this when `format` is unset, and browsers just display the plain text content when this endpoint is queried.

Would you like me to leave this as is, set it as an attachment (download automatically), or set it as inline (display in browser)? I don't feel strongly one way or another.

Reply:
> given that (unlike zip files) browsers can render this output without issue

Good point. I don't feel strongly. Totally happy to ship it as-is.